ui/analyse/src/study/relay/videoPlayerView.ts

```diff
@@ -14,8 +14,19 @@ export function renderVideoPlayer(relay: RelayCtrl): VNode | undefined {
   });
 }
 
-export function resizeVideoPlayer(): void {
-  player?.cover(document.getElementById('video-player-placeholder') ?? undefined);
+export function onWindowResize(redraw: Redraw) {
+  let cols = 0;
+  window.addEventListener(
+    'resize',
+    () => {
+      player?.cover(document.getElementById('video-player-placeholder') ?? undefined);
+      const newCols = Number(window.getComputedStyle(document.body).getPropertyValue('--cols'));
```

Comment:
Looks like I missed something. We no longer use --cols, i changed it to a --allow-video bool since abandoning the flexbox.

Reply:
I can get this in a follow up pr if you don't see the comment

Reply:
Unfortunately, I think we must still track the old state of the css variable if we want to avoid triggering the snab redraw when there's not a state change.